.github/workflows/rust.yml

```diff
@@ -0,0 +1,25 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: Swatinem/rust-cache@v2.2.0
+    - name: 🔨 Build
+      run: cargo build --verbose
+    - name: 🪲 Run tests
+      run: cargo test --verbose
+    - name: 📖 Check formatting
+      run: cargo fmt --check
```

Comment:
maybe split into 3 jobs and make job names contain emojis instead?

Reply:
i would like the emojis to be visible here 
![image](https://user-images.githubusercontent.com/72213407/205685704-d4cafa85-e821-4f1a-a8a7-b2e586e9f2e7.png)